recursive-digest/src/lib.rs

```diff
@@ -215,25 +235,13 @@ pub fn get_recursive_digest_for_dir<
 ) -> Result<Vec<u8>, DigestError> {
     let mut hasher = RecursiveDigest::<Digest>::new(root_path.into(), None);
 
-    for entry in walkdir::WalkDir::new(root_path)
-        .into_iter()
-        .filter_entry(|entry| {
```

Comment:
Ouch. This is necessary, so we can specify a directory to be ignored altogether in the ignorelist. @ThomasdenH